bt_smarthub_devicelist/btsmarthub_devicelist.py

```diff
@@ -5,16 +5,33 @@
 import random
 import hashlib
 import logging
+from bt_smarthub_devicelist import btsmarthub2_devicelist
```

Comment:
I believe you do have a typo here
it should be **from btsmarthub_devicelist import btsmarthub2_devicelist** instead :)

Reply:
I fixed this in #8. The package name was refactored after this pull request was opened.

Reply:
Oh, haven't seen that. Sorry :) looks like support for Smart Hub is happening soon